libazureinit/src/lib.rs

```diff
@@ -14,5 +15,7 @@ pub use provision::{
     Provision,
 };
 
+mod unittest;
```

Comment:
Maybe 
```suggestion
#[cfg(test)]
mod unittest;
```
so it only shows up to the tests? I'm assuming this works, but I've not tried it myself.